prqlc/prql-compiler/tests/integration/bad_error_messages.rs

```diff
@@ -90,9 +90,7 @@ fn array_instead_of_tuple() {
        │
      3 │     select [e.first_name, e.last_name]
        │            ─────────────┬─────────────
-       │                         ╰─────────────── unexpected `[this.e.first_name, this.e.last_name]`
-       │
-       │ Help: this is probably a 'bad type' error (we are working on that)
+       │                         ╰─────────────── unexpected array of values (not supported here)
```

Comment:
FWIW we could now promote this from this file to `error_messages.rs`, as it's now pretty good (but in another PR is fine)